core/controllers/editor.py

```diff
@@ -772,7 +772,8 @@ def post(self, exploration_id):
                 'A file with the name %s already exists. Please choose a '
                 'different name.' % filename)
 
-        fs.commit(self.user_id, filepath, raw, mimetype=mimetype)
+        image_services.create_compressed_versions_of_image(
```

Comment:
From the name, I would expect this to return a 2-tuple with just the compressed versions, and not to save the original file.

Could we move the "save all three files" call out of this function and into the caller? Ditto in the job.

Alternatively, call it save_original_and_compressed_versions_of_image() and move it to exp_services.

Reply:
Done